tests/scripts/test-jsonrpc-server.sh

```diff
@@ -20,19 +20,17 @@ script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
 remote_cartesi_machine=$1
 cartesi_machine=$2
 cartesi_machine_tests=$3
-test_path=$4
-lua=${5:-lua5.4}
+lua=$4
+test_path=${CARTESI_TESTS_PATH}
```

Comment:
We could automate cleanup and exit on any error (same for test-grpc-server.sh):
```
set -e
trap wait_for_shutdown EXIT
```

Reply:
thanks

Reply:
ok, created an issue for it.